blinking-Butled/README.md

```diff
@@ -0,0 +1,45 @@
+# blinking-Butled
+
+## Firmware
+To build the PRU firmware:
+
+	cd blinking-led/pru
+	make
+
+## UIO host driver
```

Comment:
If you have not tested UIO, then please remove this section and the relevant host-side code.